src/gui_common/RadialPopup.cs

```diff
@@ -55,6 +55,12 @@ public bool RadialPopupCanceled()
         return true;
     }
 
+    protected override void OnHidden()
+    {
+        base.OnHidden();
+        OnClosed();
```

Comment:
Yes, it makes sense to simplify.

Though, I see `OnCancelled(int itemId)` seem to take in one argument, but `EmitSignal(nameof(OnCanceled));` never pass any arguments, why is this and should the `itemId` parameter be removed?

Reply:
My guess is that it is a copy-paste mistake and it is safe to remove that unused parameter.

Reply:
The signal itself isn't used so I guess it can be just removed entirely...

Reply:
I added the signal because I foresee a need for something to know if the user cancels picking a thing from an action wheel. Maybe with nested action wheels or maybe in the future the action wheel would be used in a modal way so that we need to know if the action is cancelled to clean up other state.

Reply:
Keeping this would be redundant anyway since there's already the built-in CanvasItem's `hide` signal.

Reply:
I guess sort of, but I think the intention is much more clear when the "user cancelled picking something" is communicated clearly. Going back to my multi level action wheel example, that *might* get implemented so that the first wheel is hidden when the top level item is selected. And that would mean that using the generic hidden signal would not work and instead the domain specific signal would be much superior.